libs/wordpress-plugin/plugin/trunk/block/edit-or-preview/edit.tsx

```diff
@@ -52,6 +54,19 @@ export const Edit = ({
   const [entitySubgraph, setEntitySubgraph] =
     useState<Subgraph<EntityRootType> | null>(null);
 
+  const displayToast = useCallback(
+    (toastProps: ToastProps, options?: Parameters<typeof toast>[1]) => {
+      toast(<Toast {...toastProps} />, {
```

Comment:
The action text is green, is this intended? It looks slightly out of place
![image](https://user-images.githubusercontent.com/6988668/225671114-03989349-fd2d-4de3-9387-7fbc34cf734a.png)

Reply:
I have updated the toast styling in https://github.com/blockprotocol/blockprotocol/pull/1181/commits/081b445281c6c7138ef94d708668db11fd325b31 – please see new demo in description